web-console/knikubevirt/windows-vms/windows-vms.md

```diff
@@ -0,0 +1,66 @@
+
+## Create VM flow  
+
+1. When the user chooses windows OS from the Operating system drop down menu, they are notified that the Windows guest tools will be mounted which will then show in the Advanced step > virtual hardware (maybe we should label this as a CD-ROM?)
+
+![user chooses windows OS from the Operating system drop down menu](img/General1.png)
+
+![user is notified that Windows guest tools will be mounted](img/General2advanced.png)
+
+2. Storage and Networking steps
+
+The main issue here is the drivers for the interface: The user needs Sata interface to set the VM but once they install the drivers they can switch to the Virtio-scsi interface.
+
+So at first, since the interface would be prepopulated with IDE, the user will see the Sata interface (because that is the one needed to install Windows).
+
+When a Windows VM is chosen, the disk interface will not be able to be Virtio-scsi because the VM will not have the drivers for it, so we need to advise the user to install the drivers after the VM has been created and then switch the interface from IDE to Virtio-scsi (to do this switch they will need to power off the VM).
+The drivers needed for Virtio-scsi are available via the Windows Guest Agent which is available by default.
+
+### Networking step
+
+![user is notified about the drivers](img/Networking1.png)
+
+### Storage step
+
+![user is notified about the drivers](img/Storage1.png)
+
+3. The user starts up the VM, then goes into Windows and installs the drivers.
+
+4. In the list view the user sees that the VM is running but that they should now switch the interface to Virtio-scsi for better performance.
+
+5. In the status popover they are provided with a “Switch interface” action. They will also be notified that the VM must be restarted for the driver update to take effect.
+
+## Post creation steps
+
+The users would need to be aware that switching the interface for the disk is useful (but not required). It’s important to note that it’s not required because they will not get an error stating there is an issue.
+
+It seems reasonable to have a notification in the Storage step that advises them to make the update and then in the VM list to show that same notification under the status column.
+
+![user is notified that something is going on with this VM](img/Notification1.png)
+
+By click a popover will show and provide an explanation
+
+![Popover with explanation](img/NotificationPopover.png)
+
+If they choose to switch Interfaces they will be asked when they choose to schedule it
+
+![user can schedule to switch interface](img/SwitchInterface.png)
+
+### CD-ROMs
+
+The user will see these guest tools in the VM details, under the overview tab in the ‘CD-ROM(s)’ section, and have the indication that Windows Drivers are installed.
```

Comment:
I don't believe this would actually indicate the drivers are installed just that the guest tools are avialable via a CD-ROM in windows. I wonder if we should indicate at the VM Details level that the drivers have been installed and the CD can be ejected. @jelkosz can we detect this?